internal/cli/atlas_alerts_config_list.go

```diff
@@ -51,21 +52,21 @@ func (opts *atlasAlertConfigListOpts) Run() error {
 	return json.PrettyPrint(result)
 }
 
-func (opts *atlasAlertConfigListOpts) newListOptions() *atlas.ListOptions {
+func (opts *atlasAlertsConfigListOpts) newListOptions() *atlas.ListOptions {
 	return &atlas.ListOptions{
 		PageNum:      opts.pageNum,
 		ItemsPerPage: opts.itemsPerPage,
 	}
 }
 
 // mongocli atlas alerts config(s) list --projectId projectId [--page N] [--limit N]
-func AtlasAlertConfigListBuilder() *cobra.Command {
-	opts := &atlasAlertConfigListOpts{
+func AtlasAlertsConfigListBuilder() *cobra.Command {
+	opts := &atlasAlertsConfigListOpts{
 		globalOpts: newGlobalOpts(),
 	}
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List alert configurations for a project.",
+		Short:   description.ListConfig,
```

Comment:
We have multiple lists so this makes the point we need this to be
```suggestion
		Short:   description.AlertsConfigList,
```